.github/workflows/docs.yml

```
@@ -0,0 +1,33 @@
+name: Publish docs to GitHub Pages
+
+on:
+  push:
+    branches:
+      - main
+  pull_request: # TODO: Before merging, stop running this action on PRs.
```

```suggestion
  pull_request: # TODO: Before merging, stop running this action on PRs.
```

Just a reminder that this was TODO.